pallets/ddc-verification/src/lib.rs

```diff
@@ -1821,10 +1844,10 @@ pub mod pallet {
 				calculated_merkle_root
 			);
 
-			let traverse_response = Self::_fetch_traverse_response(
+			let root_merkle_node = Self::_fetch_traverse_response(
```

Comment:
If we enabled the Challenging, let's maybe remove the `_` prefix from all previously unused functions such as `_challenge_aggregate_proto`, `_fetch_traverse_response`, `_find_random_merkle_node_ids` and all the rest ?

Reply:
`_fetch_traverse_response` is still unused, because the `*_proto` version does not check merkle proofs at the moment, but `_*` prefix removed from the rest of the functions which currently in use.